user-dashboard.css

```diff
@@ -0,0 +1,189 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800;900&display=swap');
+
+body {
+    background-color: rgba(169, 175, 197, 0.05);
+    font-family: Nunito;
+}
+
+.bg-blue {
+    background-color: #1F59BB;
+}
+
+.dashboard-wrap {
+    display: flex;
+    justify-content: space-between;
+}
+
+.dashboard-wrap > nav {
+    background-color: #1F59BB;
+    text-align: center;
+    max-width: 138px;
+}
+
+.dashboard-content {
+    margin: 0 auto;
+}
+
+.dashboard-wrap  nav div {
+    text-align: center;
+}
+
+.top-bar {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.top-bar nav .input-group .input-group-append > button:hover {
+    background: #1F59BB;
+    opacity: .9;
+}
+
+.top-bar .display-profile {
+    display: flex;
+}
+
+.top-bar .display-picture {
+    border-radius: 200px;
+    width: 50px;
+    height: 50px;
+    background: #fff;
+}
+
+
+.dashboard-content .profile-summary {
+    background-image: url('img/svgs/profile-summary-bg.svg');
+    background-size: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5690830)